doc/Requirements Specification/Parts/Terms and Definitions.lyx

```diff
@@ -1359,6 +1359,54 @@ glspl{resource demand}.
 }
 \end_layout
 
+\begin_layout Plain Layout
+
+\end_layout
+
+\begin_layout Plain Layout
+
+
+\backslash
+newacronym{JRE}{JRE}{
+\backslash
+gls{Java Runtime Environment}}
+\end_layout
+
+\begin_layout Plain Layout
+
+
+\backslash
+newglossaryentry{Java Runtime Environment}{
+\end_layout
+
+\begin_layout Plain Layout
+
+	name={Java Runtime Environment},
+\end_layout
+
+\begin_layout Plain Layout
+
+	description={
+\end_layout
+
+\begin_layout Plain Layout
+
+		a software set containing a Java Virtual Machine, a browser plugin, the
```

Comment:
It probably doesn't count as a full JRE, then. Not my idea, I just looked up what it contains and wrote it down. I didn't find it on the official website, but it says pretty much the same: https://java.com/en/download/faq/whatis_java.xml